pyiode/iode_database/database.pxd

```diff
@@ -1,5 +1,7 @@
 # distutils: language = c++
 
+# GDM> rename file to something like kdb_database or kdb or cpp_api_database or ... to make it clearer
```

Comment:
Actually, `KDB` means `Kaa DataBase`. `Kaa` being the ancestor of `IODE`. I prefere to rename the present file as cpp_api_database.pxd

Reply:
changed in PR #554 -> cpp_api_database.pxd